tests/zfs-tests/tests/functional/cli_root/zfs_destroy/zfs_destroy_common.kshlib

```diff
@@ -61,7 +61,11 @@ function setup_testenv #[dtst]
 		log_must zfs create -V $VOLSIZE $VOL
 		block_device_wait
 
-		echo "y" | newfs $ZVOL_DEVDIR/$VOL > /dev/null 2>&1
+		if is_freebsd; then
+			newfs $ZVOL_DEVDIR/$VOL > /dev/null 2>&1
+		else
+			echo "y" | newfs $ZVOL_DEVDIR/$VOL > /dev/null 2>&1
+		fi
```

Comment:
Not critical, but we could probably write this is a way which works on all platforms.  Or at worst introduce another convenience wrapper function.